launch/shell.go

```diff
@@ -34,10 +35,14 @@ func (l *Launcher) launchWithShell(self string, proc Process) error {
 	if err != nil {
 		return err
 	}
+	command := ""
+	if len(proc.Command) > 0 {
```

Comment:
For now, this is always false, right?

Reply:
I believe it is always true. You must have at least a single command (and right now only 1 command).

Reply:
Ah yes, my bad. I meant it's always of length 1.